ArduCopter/mode_throw.cpp

```diff
@@ -272,8 +272,22 @@ bool ModeThrow::throw_detected()
     // Check if the accel length is < 1.0g indicating that any throw action is complete and the copter has been released
     bool no_throw_action = copter.ins.get_accel().length() < 1.0f * GRAVITY_MSS;
 
-    // High velocity or free-fall combined with increasing height indicate a possible air-drop or throw release
-    bool possible_throw_detected = (free_falling || high_speed) && changing_height && no_throw_action;
+    // fetch the altitude above home
+    float altitude_above_home;  // Use altitude above home if it is set, otherwise relative to EKF origin
+    if (ahrs.home_is_set()) {
+        ahrs.get_relative_position_D_home(altitude_above_home);
+        altitude_above_home = -altitude_above_home; // altitude above home is returned as negative
+    }
```

Comment:
nitpick: else should be one line higher right after the }

Reply:
Done :)